crystalfontz/client.py

```diff
@@ -51,6 +51,14 @@ async def main():
 )
 import warnings
 
+try:
+    from asyncio import timeout as async_timeout
+    from asyncio import TimeoutError
```

Comment:
This import is deprecated in 3.13. I'm not sure when that happened.

This is the only file that needs to pull in both `timeout` and `TimeoutError`, and I *suspect* the issue is that only one of these imports is failing on 3.13.

Reply:
This doesn't necessarily appear to be the issue. Keep in mind that the problem isn't that the timeout doesn't get *caught*, it's that the API call hangs indefinitely.